docker-compose/docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
 
   # WHISK CONTROLLER
   controller:
-    image: whisk/controller:latest
+    image: openwhisk/controller:6a56c45
```

Comment:
I pick latest
Can someone help out putting together a jenkinsfile, we can run a jenkins job everytime openwhisk core changes to make sure docker-compose use case still works.
If it fails will send a notification to Slack.

I already have a Jenkins Job to build the images and push them, we just need another one to test docker-compose use case that it's always is working using latest images.